src/Misc/layoutroot/run.sh

```diff
@@ -9,11 +9,22 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
     [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
 done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+
 # run the helper process which keep the listener alive
 while :;
 do
+    # Set job control
+    set -m
+    # On SIGINT or SIGTERM, send SIGINT to Runner.Listener
+    trap "pkill -2 Runner.Listener" INT TERM
```

Comment:
should we do similar to the ./runsvc.sh which is using `trap 'kill -INT $PID' TERM INT`?

You might have multiple `Runner.Listener` running on the machine

Reply:
Applied